named-tunnel-k8s/README.md

```diff
@@ -1,66 +1,63 @@
-    $ cloudflared tunnel login
+```sh
+$ cloudflared tunnel login
 ```
 
-2. Create a tunnel, change example to the name you want to assign to your tunnel.
+2. Create a tunnel, change `example-tunnel` to the name you want to assign to your tunnel.
 
-```
-    $ cloudflared tunnel create example
-    INFO[2020-09-05T10:48:34+01:00] Writing tunnel credentials to /Users/cf000197/.cloudflared/ef824aef-7557-4b41-a398-4684585177ad.json. cloudflared chose this file based on where your origin certificate was found.
-    INFO[2020-09-05T10:48:34+01:00] Keep this file secret. To revoke these credentials, delete the tunnel.
-    INFO[2020-09-05T10:48:34+01:00] Created tunnel example with id ef824aef-7557-4b41-a398-4684585177ad
+```sh
+$ cloudflared tunnel create example-tunnel
+INFO[2020-09-05T10:48:34+01:00] Writing tunnel credentials to /Users/cf000197/.cloudflared/ef824aef-7557-4b41-a398-4684585177ad.json. cloudflared chose this file based on where your origin certificate was found.
+INFO[2020-09-05T10:48:34+01:00] Keep this file secret. To revoke these credentials, delete the tunnel.
+INFO[2020-09-05T10:48:34+01:00] Created tunnel example-tunnel with id ef824aef-7557-4b41-a398-4684585177ad
 ```
 
-3. Upload the tunnel credentials file to your kubernetes as a secret. Go to the directory where the credentials is saved. 
+3. Upload the tunnel credentials file to your Kubernetes as a secret. You'll need to provide the filepath that the tunnel credentials file was created under. You can find that path in the output of `cloudflared tunnel create` above.
 
-```
-    $ kubectl create secret generic tunnel-credentials --from-file=ef824aef-7557-4b41-a398-4684585177ad.json 
-    secret/tunnel-credentials created
+```sh
+$ kubectl create secret generic tunnel-credentials \
+--from-file=credentials.json=/Users/cf000197/.cloudflared/ef824aef-7557-4b41-a398-4684585177ad.json
 ```
 
 4. Associate your Tunnel with a DNS record. Go to dashboard and create a CNAME targeting <tunnel ID>.cfargotunnel.com. In this example the tunnel ID is `ef824aef-7557-4b41-a398-4684585177ad`, so I will create CNAME targeting `ef824aef-7557-4b41-a398-4684585177ad.cfargotunnel.com`. You can create multiple CNAME records targeting the same tunnel.
 
-    ![create CNAME](create-cname.png) 
+    ![create CNAME](create-cname.png)
 
-5. Deploy cloudflared as a sidecar to your application container. In this example our application is an Nginx server. 
+You can do this from the command line by running `cloudflared route dns <tunnel> <hostname>`. For example, `cloudflared tunnel route dns example-tunnel tunnel.example.com`. You can use a similar method to route traffic to `cloudflared` from a [Cloudflare Load Balancer](https://www.cloudflare.com/load-balancing/), see [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/routing-to-tunnel/lb) for details.
 
-```
-    $ kubectl apply -f deployment.yaml
-    deployment.apps/named-tunnel-k8s configured
+5. Deploy cloudflared
```

Comment:
We should explain how cloudflared deployment fits with the rest of their cluster